statistics/fmsketch.go

```diff
@@ -96,6 +96,24 @@ func (s *FMSketch) InsertValue(sc *stmtctx.StatementContext, value types.Datum)
 	return nil
 }
 
+// InsertRowValue inserts multi-column values to the sketch.
+func (s *FMSketch) InsertRowValue(sc *stmtctx.StatementContext, values []types.Datum) error {
+	b := make([]byte, 0, 8)
+	for _, v := range values {
+		b, err := codec.EncodeValue(sc, b, v)
+		if err != nil {
+			return err
+		}
+		s.hashFunc.Reset()
+		_, err = s.hashFunc.Write(b)
```

Comment:
Why do we reset `hashFunc` for each value?

Reply:
Seems a mistake. Let me change it and add tests.